### PR TITLE
RH - Adding Associations to Models

### PR DESCRIPTION
**Pull request by rezahusain**
Hey guys,

So I went through each model and added all the necessary associations with the help of the copilot. I added each association diligently, however if you guys find some kind of issue whether its with the way I'm naming the associations or anything like that, please leave a comment and I'll make the change ASAP.

### CODE REVIEW

File: app/models/review.rb
Comment (brvarner) [inline app/models/review.rb:1]: I think a review can only belong to one meeting, right?

File: app/models/program.rb
[the remark this answers is not in the document]
Comment (MayaS1111) [reply app/models/program.rb:1]: agreed

File: app/models/review.rb
Comment (MayaS1111) [reply app/models/review.rb:1]: Correct

File: app/models/user.rb [hunk 1]
```diff
@@ -28,6 +28,12 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  has_many :cohorts, class_name: 'Cohort', foreign_key: 'creator_id', dependent: :destroy
-  has_many :programs, class_name: 'Program', foreign_key: 'creator_id', dependent: :destroy
+  has_many :cohort_members, class_name: "CohortMember", foreign_key: "user_id", dependent: :destroy
```
Comment (MayaS1111) [inline app/models/user.rb:31]: How would this one be used on user? There is another cohort_members in cohort that is used differently. I guess the name is confusing me or I just don't understand it 😂.

I guess my question is what will it return?

File: app/models/user.rb [hunk 2]
```diff
@@ -28,6 +28,12 @@ class User < ApplicationRecord
...
+  has_many :mentors, class_name: "Match", foreign_key: "mentor_id", dependent: :destroy
```
Comment (MayaS1111) [inline app/models/user.rb:32]: Can a user have many mentors? Will we allow a mentee to have one that one at a time?

**Review by MayaS1111 — CHANGES_REQUESTED** [review]

**Review by brvarner — APPROVED** [review]
I think it looks good, very minor note: I'd say change line 32 in the user file to say `mentor` instead of `mentors`, because it's singular. Otherwise, great work!

**Review by MayaS1111 — APPROVED** [review]

**Review by MayaS1111 — APPROVED** [review]
Goooood to go!

**Pull request merged**